pandas/tests/groupby/aggregate/test_other.py

```diff
@@ -638,3 +638,12 @@ def weird_func(x):
 
     result = df["decimals"].groupby(df["id1"]).agg(weird_func)
     tm.assert_series_equal(result, expected, check_names=False)
+
+
+@pytest.mark.parametrize("func", ["first", "last", "max", "min"])
+def test_min_count_implementation_min_max_first_last(func):
```

Comment:
aggregate is for testing the agg method on groupby, this belongs in missing. As an aside, I'm not seeing much in the way of `min_count` tests outside of categorical. How about name this as `test_min_count` and can extend the scope later (okay to do it here too).

Reply:
Thx, moved and renamed it